lua/lvim/core/terminal.lua

```diff
@@ -148,14 +148,18 @@ end
 
 M.lazygit_toggle = function()
   local Terminal = require("toggleterm.terminal").Terminal
+
+  local columns = vim.o.columns
+  local lines = vim.o.lines
+
   local lazygit = Terminal:new {
     cmd = "lazygit",
     hidden = true,
     direction = "float",
     float_opts = {
       border = "none",
-      width = 100000,
-      height = 100000,
+      width = columns or 300,
+      height = lines or 200,
```

Comment:
using `or` doesn't do anything, since `vim.o.columns`  value is always has a value.